scrapy/extensions/feedexport.py

```diff
@@ -154,6 +154,8 @@ def __init__(self, settings):
         self.export_fields = settings.getlist('FEED_EXPORT_FIELDS') or None
         uripar = settings['FEED_URI_PARAMS']
         self._uripar = load_object(uripar) if uripar else lambda x, y: None
+        self.batch_size = settings['FEED_BATCH_SIZE']
+        self.start = datetime.utcnow().replace(microsecond=0).isoformat().replace(':', '-')
```

Comment:
(because from our docs it is not obvious templated folders are created on a local file system)